Samples/WindowsAppSDKTelemetry/cpp/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.ProjectReunion" version="0.8.1" targetFramework="native" />
+  <package id="Microsoft.ProjectReunion.DWrite" version="0.8.1" targetFramework="native" />
+  <package id="Microsoft.ProjectReunion.Foundation" version="0.8.1" targetFramework="native" />
+  <package id="Microsoft.ProjectReunion.InteractiveExperiences" version="0.8.1" targetFramework="native" />
+  <package id="Microsoft.ProjectReunion.WinUI" version="0.8.1" targetFramework="native" />
+  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.210204.1" targetFramework="native" />

```

Comment:
Why is this needed when this is already part of reunion package?

Reply:
WIL is not referenced by default in WindowsAppSDK yet. It may be present in the official build. Currently, the nightly does not contain the reference.